qtrees/data_processor.py

```diff
@@ -0,0 +1,311 @@
+import pandas as pd
+import numpy as np
+from functools import reduce
+import datetime as dt
+from typing import Union, Optional
+from sklearn.preprocessing import OrdinalEncoder
+
+ROLLING_WINDOW = 7
```

Comment:
die drei könnten
a) auch in die Config
b) ggf. auch für Nowcast und Forecast verschieden sein?

Wenn, dann ggf. wie die Hyperparameter des Random Forest behandeln und in ein Dict "NOWCAST_PREPROCESSING_HYPERPARAMETER" (oder kürzer). Weil sind ja quasi die Hyperparameter des Preprocessing.

Reply:
done